assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml

```diff
@@ -262,7 +262,7 @@ jobs:
 
   image_classification_runtime_component:
     type: command
-    component: azureml:train_image_classification_model:0.0.4
+    component: azureml:train_image_classification_model:0.0.5
```

Comment:
don't we need to update the version of this component as well?

Reply:
I already updated to version 0.0.5. Could you please check the latest commits.

Reply:
Sorry.. I meant the pipeline component version - https://github.com/Azure/azureml-assets/blob/ee3af030d0d48ef48b23f01efb0b27c4a878b51c/assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml#L4C1-L4C15

Reply:
This version was updated by Shubham to 0.0.8 along with fintune components, but this version was never released. So, using 0.0.8 for the next release.